.ci/azure-pipelines.yml

```diff
@@ -275,7 +235,9 @@ jobs:
     pool:
       vmImage: ubuntu-latest
     dependsOn: main_build
-    condition: and(succeeded(), variables['System.PullRequest.PullRequestNumber']) # Only execute if the pullrequest numer is defined. (So not for normal CI builds)
+    # disabled until ABI is expected to remain unchanged
+    # only execute for pull requests
+    condition: and(eq(1, 2), succeeded(), variables['System.PullRequest.PullRequestNumber'])
```

Comment:
Lets not disable this, but I can update the tool to have a "no-error" flag.

I'll update it this evening here: https://github.com/EraYaN/dotnet-compatibility